README.md

```diff
@@ -1,24 +1,29 @@
 # Cloud Data Quality Engine
 
-[![beta](https://badgen.net/badge/status/beta/1E90FF)](https://badgen.net/badge/status/beta/1E90FF)
+[![GA](https://badgen.net/badge/status/GA/1E90FF)](https://badgen.net/badge/status/GA/1E90FF)
 [![build-test status](https://github.com/GoogleCloudPlatform/cloud-data-quality/actions/workflows/build-test.yml/badge.svg)](https://github.com/GoogleCloudPlatform/cloud-data-quality/actions/workflows/build-test.yml)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 
 ## Introductions
 
-`CloudDQ` is a cloud-native, declarative, and scalable Data Quality validation Command-Line Interface (CLI) application for Google BigQuery.
+CloudDQ is a cloud-native, declarative, and scalable Data Quality validation Command-Line Interface (CLI) application for Google BigQuery. CloudDQ allows users to define and schedule custom Data Quality checks across their BigQuery tables. Data Quality validation results will be available in another BigQuery table of their choice. Users can then build dashboards to routinely monitor data quality from the dashboards.
```

Comment:
s/can then build dashboards/can then build dashboards or consume data quality outputs programmatically

s/monitor data quality from the dashboards/monitor data quality from the dashboards and data pipelines

Reply:
incorporated the suggestions